package.json

```diff
@@ -19,6 +19,7 @@
     "docs": "parcel build ./src/index.html ./src/examples/*.html -d docs --public-url .",
     "lint": "stylelint \"./src/**/*.css\"",
     "lint:fix": "npm run lint -- --fix",
+    "lint:js": "eslint --ext ./src/**/*.ts",
```

Comment:
Maybe `lint:scripts` instead of `lint:js`, as it's actually only targeting TypeScript at the moment? It also perhaps might be extended to target JavaScript too later.

Reply:
Yes, I think `lint:scripts` would be be better.